incubator/kylo/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+name: kylo
+version: 0.0.1
+appVersion: 0.9.0
+description: A Helm chart that installs the Kylo Data Analytics Platform.
+home: https://kylo.io/
+keywords:
+- analytics
+- data
+sources:
+  - https://github.com/Teradata/kylo
+maintainers:
+- name: William Hearn
```

Comment:
The maintainers names for this repo needs to be a github username. This allows us to reference the name in issues and PRs.